dev/NumberBox/NumberBox.xaml

```diff
@@ -165,25 +238,40 @@
                             </Grid>
                         </Popup>
 
+                        <Button x:Name="InputEater"
```

Comment:
If you hover over the inline buttons and slowly move to the left, at some point this will eat the hover input and it seems that nothing is receiving the hover. Do we need to size this better?

Reply:
I left a 4px border around the buttons, so there is an area where nothing is getting hover between the buttons, but I can trim it on the left side so that there's no dead space there.

Reply:
There are aso small regions above, below, to the right and between those buttons that have the same effect. We hardly can fix the issue with the area between the split buttons without having two buttons right? I think it is fine right now.